### ISSUE TITLE
Players cannot see or use the new damage application card

### ISSUE THREAD

**Issue by maxobremer** [foundryvtt/dnd5e#3300]
As it says in the title, players cant see the damage application card, making applying damage to their own tokens more difficult. it would be nice to have it as an option to see it and to apply damage to targeted tokens. this would reduce the burden on the dm and give some more control to players
![image](https://github.com/foundryvtt/dnd5e/assets/86924581/133fcdb1-32ad-408a-8187-30e856ff19ed)
player left, gm right

**Comment by maxobremer** [comment]
also it would be nice if you could disable the damage card. I'm sure people put a lot of effort into the feature, but for me it felt disruptive. it messed up the positioning of chat and as someone that uses RSR it felt inferior because i couldn't apply each damage type separately so it messed up a couple of turns in the session I just ran (example was the wither and bloom which i have roll damage and the healing with the same item card).
compared to RSR the only upside is the ability to ignore resistances, but in my opinion that should be handled by a character flag anyway, as features that ignore resistances, often do so for all damage of a certain type. As such I'd love to see such a thing as either a character flag or on items themselves.

still great work to those that worked on v3.1 but there certainly are kinks to work out. looking forward to what comes next.

**Comment by krbz999** [comment]
Players want to manage their own character, and GMs want to manage just their own NPCs. I think, for many, this is the way things should be.

Without the damage tray available to players, and with _that implied general assumption that the GM is the only one to apply damage to all actors_, regardless of player ownership, sadly this results in a worse experience overall when engaging in combat.

Either we go by this assumption, and players do not manage their own HP when targeted by a damage roll. They are then less in control of their own character, and furthermore they no doubt have to monitor their own HP regardless to make sure the GM, the only arbiter, remembered to apply the damage.

The flow of combat then goes like so.
- The GM rolls damage against a player character.
- The player just sits there and hopes the GM applies it.
- The GM forgets, and the player waits patiently or constantly has to interrupt the flow of the game to remind the GM.
- Everyone is frustrated because of constant interruptions.

Alternatively, the GM remembers and applies the damage.
- The player interrupts to remind the GM they have a reaction that can be used to halve the damage or ignore it entirely.
- Another player interrupts the GM to shout that they have a reaction that can reduce the damage.
- A third player interrupts the GM to shout that their aura applies resistance.
- The GM is frustrated because they, as the only person, are getting all the interruptions and have to manage all the relevant player features on top of managing their own half of the combatants.

Or we ignore this general assumption; the damage tray is available for players but allows only damage application to actors that the player owns and whose tokens they can control.

The flow of combat then goes like so.
- The GM rolls damage against a player character.
- The player applies it with the click of a button.
- Other players might interrupt the player applying the damage to point out reactions and auras.
- Meanwhile, the GM has continued on with their half of the combatants in this combat with no interruptions.

Last option is the GM makes the choice that, despite the damage tray (which is not available to players) calculating damage correctly with regards to resistances, vulnerabilities, immunities, and modifications, they will not apply damage to player-owned actors. I don't feel this option is even worth discussing.

### Last thoughts
The only thing that would non-trivialize what I would consider the ideal implementation is the flow of selecting applicable tokens for damage application. This is only relevant for areas-of-effect like a _Fireball_ spell where a player might be inside the area with a group of enemy NPCs, in which case the GM will want to make sure not to select the player-owned token.

The list of controlled/targeted tokens could perhaps be split between player and non-player tokens, or it could receive further configuration by a setting that could be configured to appeal to a given table's playstyle (the GM applies to all / players apply to themselves / manual).

And lastly, I'm simply not going to consider the notion that a user should be able to apply damage to an actor they do not have ownership of, which I feel should be left squarely in module territory.

**Comment by BlackBadger777** [comment]
I don't know how it is for other tables, but at my table, the GM would never dream of editing the player characters' hit points on their sheets. This is basically what happens when the damage application tray is locked for GMs' use only. Especially at higher levels where resistances and damage reductions become more frequent, the damage application tray is **the** way to apply damage to actors in Foundry VTT. For me, giving players access to use the damage tray for actors they own is not only a nice qol feature, but a crucial requirement. As things stand, players' agency is severely infringed upon if the GM is expected to apply damage to their characters via the damage tray.

**Comment by nschoenwald** [comment]
> I don't know how it is for other tables, but at my table, the GM would never dream of editing the player characters' hit points on their sheets. This is basically what happens when the damage application tray is locked for GMs' use only. Especially at higher levels where resistances and damage reductions become more frequent, the damage application tray is **the** way to apply damage to actors in Foundry VTT. For me, giving players access to use the damage tray for actors they own is not only a nice qol feature, but a crucial requirement. As things stand, players' agency is severely infringed upon if the GM is expected to apply damage to their characters via the damage tray.

Agreed 100%. I can't understand the way this is currently implemented, or rather not implemented.

**Comment by nschoenwald** [comment]
@arbron Are there currently no plans to adress this issue?

**Comment by Fyorl** [comment]
Issues that we do not plan to implement are closed as 'won't change', every other issue is up for consideration. We have a backlog of over 800 issues that require sorting and prioritisation, that we must also balance with the needs of upcoming 3rd-party and 1st-party content.

I'd ask that you refrain from pinging maintainers on every issue that you have a personal vested interest in.

**Comment by vfez1** [comment]
I’ve implemented a solution via a world setting to control player visibility of Damage Tray buttons and did a Pull Request. Feedback welcome if a different approach is preferred.
